sos/cleaner/mappings/hostname_map.py

```diff
@@ -194,7 +190,10 @@ def sanitize_item(self, item):
                 ob_hostname = 'unknown'
             ob_domain = self.sanitize_domain(domain)
             self.dataset[item] = ob_domain
-            return '.'.join([ob_hostname, ob_domain])
+            _fqdn = '.'.join([ob_hostname, ob_domain])
+            if all([h.isupper() for h in host]):
+                _fqdn = _fqdn.upper()
```

Comment:
Shall not we apply this also for `len(host) == 2`? The current code does apply so, the proposed one doesn't.

Reply:
Ack, updated and pushed.